### DISCUSSION TITLE
Failed to dowload Setup Java for Flutter App

### DISCUSSION THREAD

**Discussion by ycv005** [community/community#26964] [Actions]
<p>Following is my Github Action setup- </p>
<pre><code>on:
  push:
    branches:
    - master
    - develop/*
  pull_request:
    branches:
    - master
    - develop/*

jobs:
  build:
    name: Build APK
    runs-on: ubuntu-latest

    container:
      image: google/dart:latest

    steps:
    - uses: actions/checkout@v2
    - uses: actions/setup-java@1
      with:
        java-version: '11.0.5'
    - uses: subosito/flutter-action@v1
      with:
        channel: stable
    - name: Install dependencies
      run: pub get
    - name: Run tests
      run: pub run test
    - run: flutter build apk
</code></pre>
<p>I am getting below error- </p>
<pre><code>Failed to download action 'https://api.github.com/repos/actions/setup-java/tarball/1'. Error Response status code does not indicate success: 404 (Not Found).
</code></pre>
<p>Here is the Image of Full Error -  </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f1bb481ff73bf523733a2d295db50582079cf3f6.png" data-download-href="/uploads/short-url/yusjOJK2pw7ypBt46uKZe6RflHw.png?dl=1" title="Screenshot from 2020-05-17 21-07-07.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26734819/181100983-b1cb0d8f-b601-4c2e-a347-a42d02d54708.png" alt="Screenshot from 2020-05-17 21-07-07.png" data-base62-sha1="yusjOJK2pw7ypBt46uKZe6RflHw" width="690" height="160" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f1bb481ff73bf523733a2d295db50582079cf3f6_2_690x160.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/f1bb481ff73bf523733a2d295db50582079cf3f6_2_1035x240.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f1bb481ff73bf523733a2d295db50582079cf3f6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f1bb481ff73bf523733a2d295db50582079cf3f6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-05-17 21-07-07.png</span><span class="informations">1295×302 45.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks in Advance.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/ycv005">@ycv005</a> , </p>
<p>The Setup Java action has releases with tags v1.2.0 and v1.3.0. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/28338fb6d021594b4c872e43eb5744d96c3ca2f8.png" data-download-href="/uploads/short-url/5JDzfAEsWCIy6jklDkTJPerjDw4.png?dl=1" title="setup java.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181100985-6801cdba-79ba-4d8c-a1e2-e5f54a7d1408.png" alt="setup java.png" data-base62-sha1="5JDzfAEsWCIy6jklDkTJPerjDw4" width="593" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/28338fb6d021594b4c872e43eb5744d96c3ca2f8_2_593x499.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/28338fb6d021594b4c872e43eb5744d96c3ca2f8_2_889x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/28338fb6d021594b4c872e43eb5744d96c3ca2f8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/28338fb6d021594b4c872e43eb5744d96c3ca2f8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">setup java.png</span><span class="informations">1031×868 48.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>You need to use <span class="mention">@v1</span> instead of <span class="mention">@1</span> .</p>
<pre><code>- uses: actions/setup-java@v1</code></pre>
